_scss/_utilities.scss

```diff
@@ -58,6 +58,9 @@ i.fa.fa-outdent {
     padding: 0px 15px 10px 15px;
 }
 
+.image-inline img {
+    display: inline;
```

Comment:
You might experiment with `inline-block` here. Not sure what the difference will be. I think you can put height attributes on `inline-block` things but not `inline`.